scripts/ci/run-ci-tests.sh

```diff
@@ -289,6 +289,20 @@ ip net add test
 ./test/zdtm.py run -t zdtm/static/env00 -t zdtm/transition/fork -t zdtm/static/ghost_holes00 -t zdtm/static/socket-tcp -t zdtm/static/msgque -k always
 ./test/crit-recode.py
 
+# Rootless tests
+if capsh --supports=cap_checkpoint_restore; then
+	make -C test/zdtm/ cleanout
+	rm -rf test/dump
+	setcap cap_checkpoint_restore,cap_sys_ptrace+eip criu/criu
+	# Run it as non-root in a user namespace. Since CAP_CHECKPOINT_RESTORE behaves differently in non-user namespaces (e.g. no access to map_files) this tests that we can dump and restore
```

Comment:
It is out of scope of this pr. And I am very skeptical about this especially after reading discussions around PR_HIDE_SELF_EXE. But it can be worth to send a patch and check the ground.